mne/preprocessing/eyetracking/tests/test_calibration.py

```diff
@@ -198,8 +198,8 @@ def test_read_calibration(fname):
     ]
 
     assert len(calibrations) == 2  # calibration[0] is left, calibration[1] is right
-    assert calibrations[0]["onset"] == 0
-    assert calibrations[1]["onset"] == 0
+    assert np.isclose(calibrations[0]["onset"], -6.85)
+    assert np.isclose(calibrations[1]["onset"], -6.85)
```

Comment:
Do not `assert np.isclose` or `assert np.allclose`, just use
```suggestion
    assert_allclose(calibrations[0]["onset"], -6.85)
    assert_allclose(calibrations[1]["onset"], -6.85)
```
all `assert np.array_equal` should be `assert_array_equal` etc.

Reply:
Yeah always feel free to push changes instead of committing suggestions, often for me at least the suggestion is just an easy way to express the gist of an idea

Reply:
got it!